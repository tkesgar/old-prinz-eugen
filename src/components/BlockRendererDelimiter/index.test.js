import { mount } from '@vue/test-utils'
import localVue from '../../utils/test/local-vue'
import BlockRendererDelimiter from './'

describe('BlockRendererDelimiter', () => {
  const wrapper = mount(BlockRendererDelimiter, { localVue })

  test('should match snapshot', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
})
