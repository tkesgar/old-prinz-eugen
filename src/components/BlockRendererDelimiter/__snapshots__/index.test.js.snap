// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BlockRendererDelimiter should match snapshot 1`] = `
<div
  class="delimiter text-center py-4"
/>
`;
